doc/extension.rst

```diff
@@ -18,9 +18,9 @@ dependencies.
 Usage
 -----
 
-Add ``hawkmoth`` to :data:`sphinx:extensions` in ``conf.py``. Note that
-depending on the packaging and installation directory, this may require
-adjusting the :envvar:`python:PYTHONPATH`.
+Add ``hawkmoth`` to :any:`extensions` in ``conf.py``. Note that depending on the
```

Comment:
Case in point, we actually have a reference called `extension` but we've been talking about having extensionS to hawkmoth ourselves... so it would be unsurprising if we aliased this one in the future.

I haven't tested it, but I believe intersphinx stops looking once it gets a hit, so we wouldn't even get a warning about the alias.

Reply:
Agreed.